docs/install/quick-start-guide.md

```diff
@@ -0,0 +1,150 @@
+---
+title: "Quick Start Guide"
+linkTitle: "Quick Start Guide"
+weight: 1
+---
+
+# Quick Start Guide - Non-Production
+
+This guide will provide step by step instructions on how to install Ondat onto your cluster, with our helm chart, for a non-production environment.
+
+> ⚠️ This guide is for a non-production installation. Please follow our [other installation guides](https://docs.ondat.io/docs/install/) for a production-ready installation of Ondat
```

Comment:
I would suggest that we add a note that people should sign up for the portal so they can obtain a licence

Reply:
e.g. ⚠️ Make sure to add an [Ondat licence](https://github.com/ondat/documentation/blob/main/docs/operations/licensing) after installing. You can request a licence via the [Ondat SaaS Platform](https://portal.ondat.io/).

Reply:
@Ricardo-Osorio why the thumbs down? we have it for every install guide lol

Reply:
We have a licensing section at the bottom. It links to the licensing page (which may be out of date). 

I think it's better to try and link to a single page, rather than having the same info on all pages (as it makes it easier to update). I'm happy to change it if you disagree though

Reply:
happy to leave it out for now, but I would definitely want to mention the portal in this guide